sensei-lms.php

```diff
@@ -77,6 +77,11 @@ function is_sensei_activating( $activating_plugin, $plugin ) {
 	return;
 }
 
+if ( ! Sensei_Dependency_Checker::check_assets() ) {
+	add_action( 'admin_notices', array( 'Sensei_Dependency_Checker', 'add_assets_notice' ) );
+	return;
```

Comment:
Removing this return should fix most tests. It is okay if Sensei loads, but it is a good idea to warn them about the assets.

Reply:
Got it. Yes, it is not expected behavior in this case.